### PR TITLE
Added support for windows containers

### PR DESCRIPTION
**Pull request by javier-op**
Implements #8

### CODE REVIEW

**Comment by dgkanatsios** [comment]
> 		It("should attach init container", func() {

can we add a test for Windows? It would make a difference if we (by mistake) change one of the fieldnames etc.

---
Refers to: pkg/operator/controllers/utilities_test.go:137 in 4040156. [](commit_id = 40401567af8bca4b1e80dfb38579cded2a57878c, deletion_comment = False)

File: windows/buildDockerWin.ps1 [hunk 1]
```diff
@@ -0,0 +1,27 @@
+param ($registry, $version)
```
Comment (dgkanatsios) [inline windows/buildDockerWin.ps1:1]: can we add a comment/description that this is used by the GitHub Actions workflow?

File: pkg/operator/config/manager/manager.yaml [hunk 2]
```diff
@@ -149,6 +155,54 @@ spec:
         - name: LOG_LEVEL
           value: ${LOG_LEVEL}
 ---
+apiVersion: apps/v1
```
Comment (dgkanatsios) [inline pkg/operator/config/manager/manager.yaml:158]: so we're deploying the Windows DaemonSet even if the user doesn't want it (or will never use Windows containers). Maybe we can add a note on the quickstart that users can delete it afterwards? kubectl delete -n thundernetes-system daemonset nodeagent-win

File: Makefile [hunk 3]
```diff
@@ -100,7 +104,9 @@ create-install-files-dev:
 	INSTALL_FILES_FOLDER=../../installfilesdev \
 	IMG=$(NS)$(IMAGE_NAME_OPERATOR):${IMAGE_TAG} \
 	IMAGE_NAME_INIT_CONTAINER=$(NS)$(IMAGE_NAME_INIT_CONTAINER) \
+	IMAGE_NAME_INIT_CONTAINER_WIN=$(NS)$(IMAGE_NAME_INIT_CONTAINER_WIN) \
 	IMAGE_NAME_NODE_AGENT=$(NS)$(IMAGE_NAME_NODE_AGENT) \
+	IMAGE_NAME_NODE_AGENT_WIN=$(NS)$(IMAGE_NAME_NODE_AGENT_WIN) \
```
Comment (dgkanatsios) [inline Makefile:109]: overall comment: I'm pretty sure that docs/README mention that Thundernetes is only used for Linux game servers, can we take a quick look to fix this? i.e. where we say "Linux game servers" just mention "game servers in both Windows and Linux" or something?

File: windows/buildDockerWin.ps1 (hunk 1 above)
Comment (javier-op) [reply windows/buildDockerWin.ps1:1]: It's the same script used for building the images for local development, I'll add comments about this.

File: pkg/operator/config/manager/manager.yaml (hunk 2 above)
Comment (javier-op) [reply pkg/operator/config/manager/manager.yaml:158]: Ah yeah, the pods wouldn't get deployed but the daemon would stay there, I'll add that somewhere.

**Pull request merged**